requirements-test.txt

```diff
@@ -1,6 +1,7 @@
 pytest==7.4.4 ; python_version <= '3.10'
 pytest==8.3.3 ; python_version >= '3.11'
 pytest-json-report==1.5.0
+pytest-cov==5.0.0
```

Comment:
Last version is 6.0 - why use an older one?

Reply:
Good catch, the version 6.0 was released as this PR was waiting for review. Will update.